xml2kinto/tests/test_main.py

```diff
@@ -1,8 +1,20 @@
+import codecs
+import json
 import mock
 import unittest
 
 from xml2kinto import __main__ as main
 
+SCHEMAS = {
+    'certificates': None,
+    'gfx': None,
+    'add-ons': None,
+    'plugins': None,
+}
+
+with codecs.open(main.SCHEMA_FILE, encoding='utf-8') as f:
+    SCHEMAS = json.load(f)
```

Comment:
Actually https://github.com/mozilla-services/xml2kinto/pull/34/files#diff-c17a4b480e20f1476567f04b1e1ec2f6R95 this is using it for all ``assert_arguments`` calls